package.json

```diff
@@ -32,7 +32,9 @@
     "format:js": "prettier --write \"**/*.{md,js,json}\"",
     "format:pkg": "./lib/cli/index.js -wp",
     "lint": "eslint \"**/*.js\"",
-    "test": "jest"
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "test:update-snapshot": "jest --updateSnapshot --color=false"
```

Comment:
Useful run scripts for dev.

Reply:
these are unnecessary since they can be accomplished by running `yarn test --coverage` and `yarn test -u`

Reply:
@camacho Removed.